hera_qm/xrfi.py

```diff
@@ -1642,7 +1642,8 @@ def xrfi_run(ocalfits_file, acalfits_file, model_file, data_file, history,
 def day_threshold_run(data_files, history, nsig_f=7., nsig_t=7.,
                       nsig_f_adj=3., nsig_t_adj=3., clobber=False,
                       run_check=True, check_extra=True,
-                      run_check_acceptability=True):
+                      run_check_acceptability=True, separable_flags=False,
+                      time_threshold=0.15, freq_threshold=0.01):
```

Comment:
It's minor, but could you put the new parameters in front of the check keywords, for consistency across the module?